docs/software-development/index.md

```diff
@@ -19,15 +21,22 @@ Lorem ipsum dolor sit amet consectetur adipiscing elit. Dolor sit amet consectet
+
+| Product category | Development category | Name       | Description                                                                                                                                                                                                                                                                                                                                                       |
+|:-----------------|----------------------|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 1                | 1                    | Undefined  | Undefined level, no policies or procedures established for the software process. Ad hoc unpredictable software development. Poorly controlled and reactive. Success is effort of individuals (local heroes).                                                                                                                                                      |
+| 2                | 2                    | Repeatable | Repeatable level, basic project management established for software development (to track cost, functionality, and time). Development process decisions made are often reactive and based on intuition or experience instead of executing a predefined plan. The process is at least documented sufficiently such that repeating the same steps may be attempted. |
+| 3                | 3                    | Mature     | Mature level. Software development process for management and engineering is documented. At this level, the software development process is defined clearly while at organizational level the processes are not standardized.                                                                                                                                     |
+| 4                | 4                    | Defined    | Defined level. Software development process for management and engineering are integrated in the organization. Projects tailor their processes from organization’s standards. The processes are qualitative measured and scored.                                                                                                                                  |
+| 4                | 5                    | Managed    | Managed level. The quality of the software process is quantitatively measured, so it can be evaluated and adjusted when necessary.                                                                                                                                                                                                                                |
```

Comment:
Remove 5 and 6 they are not applicable for QuTech but I added them for completeness.